BlocSpot/BlocSpot/LocationManager.h

```diff
@@ -0,0 +1,28 @@
+//
+//  LocationManager.h
+//  BlocSpot
+//
+//  Created by McCay Barnes on 8/6/15.
+//  Copyright (c) 2015 McCay Barnes. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+@class LocationManager;
+
+@protocol LocationManagerDelegate <NSObject>
```

Comment:
That looks fine but do you implement the LocationManagerDelegate in the view controllers?